### PR TITLE
HTML runner fixes

### PR DESCRIPTION
**Pull request by gzm0**

### CODE REVIEW

**Comment by gzm0** [comment]
Needs rebase on "towards 0.6.10" commit. Review by @sjrd

File: test-suite/js/src/test/scala/org/scalajs/testsuite/utils/HTMLRunner.scala [hunk 1]
```diff
@@ -105,16 +108,26 @@ object HTMLRunner extends js.JSApp {
...
+      scrollDown()
     }
 
     def hr(): Unit = {
       val elem = dom.document.createElement("hr")
       dom.document.body.appendChild(elem)
     }
+
+    private def scrollDown() =
+      dom.window.scrollTo(0, dom.document.body.scrollHeight)
```
Comment (sjrd) [inline test-suite/js/src/test/scala/org/scalajs/testsuite/utils/HTMLRunner.scala]: Sounds like the commit message should appear as a comment here.

File: test-suite/js/src/test/scala/org/scalajs/testsuite/utils/HTMLRunner.scala [hunk 2]
```diff
@@ -127,6 +140,7 @@ object HTMLRunner extends js.JSApp {
       def appendChild(child: Element): Unit = js.native
       def setAttribute(name: String, value: String): Unit = js.native
       var textContent: String = js.native
+      val scrollHeight: Int = js.native
```
Comment (sjrd) [inline test-suite/js/src/test/scala/org/scalajs/testsuite/utils/HTMLRunner.scala]: This should be a `def`, not a `val`, as its value can definitely change over time.

**Comment by sjrd** [comment]
That's all.

**Comment by gzm0** [comment]
Updated.

**Comment by sjrd** [comment]
LGTM

**Comment by sjrd** [comment]
This can now be rebased on top of master.

**Comment by gzm0** [comment]
Rebased

**Pull request merged**